specification/sql/resource-manager/Microsoft.Sql/preview/2022-11-01-preview/ManagedDatabaseSensitivityLabels.json

```diff
@@ -360,18 +348,7 @@
             "type": "string"
           },
           {
-            "name": "sensitivityLabelSource",
-            "in": "path",
-            "description": "The source of the sensitivity label.",
-            "required": true,
-            "type": "string",
-            "enum": [
-              "current"
-            ],
-            "x-ms-enum": {
-              "name": "WritableSensitivityLabelSource",
```

Comment:
this enum name is not the same type as last one. why do we manually change it? 

all our swagger files are auto generated. please contact feature owner to fix it.